Svc/FileUplink/FileUplink.cpp

```diff
@@ -174,6 +185,18 @@ namespace Svc {
     this->m_lastSequenceIndex = sequenceIndex;
   }
 
+  bool FileUplink ::
+    checkDuplicatedPacket(const U32 sequenceIndex)
+  {
+    // check for duplicate packet
+    if (sequenceIndex == this->m_lastSequenceIndex) {
+      this->m_warnings.packetDuplicate(sequenceIndex);
```

Comment:
How do you avoid a duplicate on the first packet if the first packet sequence number is 0?

Reply:
Fair point. I suppose `m_lastSequenceIndex` should be initialized to some non-zero value that also won't be a valid packet index (eg. `U32MAX`).

Reply:
I think it would actually drop the first packet, since the last status was OK.

Reply:
You're right; `m_lastPacketWriteStatus` should be initialized to something other than `Os::File::OP_OK`.

Reply:
Whatever solution you come up with, you should try running it against the ground system and uplink a file nominally to make sure something doesn't get through.